damus/Util/Images/ImageSaver.swift

```diff
@@ -0,0 +1,19 @@
+// ImageSaver.swift
+
+import UIKit
+
+class ImageSaver: NSObject, ObservableObject {
```

Comment:
Swift classes don't need to be subclasses of `NSObject` (unless used from Objective-C). And since this class doesn't have a subclass, I'd mark it with `final`, which allows the compiler to optimize things and makes it easier to debug in the future:
```swift
final class ImageSaver: ObservableObject { ... }
```

Reply:
Oh, dang. You're right. My bad.